staging/src/k8s.io/api/apps/v1beta1/types.go

```diff
@@ -420,6 +420,10 @@ type DeploymentSpec struct {
 	Selector *metav1.LabelSelector `json:"selector,omitempty" protobuf:"bytes,2,opt,name=selector"`
 
 	// Template describes the pods that will be created.
+	// When the RestartPolicy field is specified in the Pod Template for Deployment,
+	// only a .spec.template.spec.restartPolicy equal to Always is allowed,
+	// which is the default if not specified.
+	// The only allowed template.spec.restartPolicy value is "Always".
```

Comment:
simplify to `The only allowed template.spec.restartPolicy value is "Always".`

Reply:
😅 I missed it. Updated. Thanks.